cmake/std/PullRequestLinuxDriverTest.py

```diff
@@ -132,8 +142,15 @@ def confirmGitVersion():
     return None
 
 
-
 def setBuildEnviron(arguments):
+    """
+    TODO: Needs explanation.
+          - How is environMap structured?
+          - How is moduleMap structured?
+    TODO: wcm: Could it be possible to offload environMap and moduleMap into a
+          JSON or Yaml file?  (Yaml allows comments which is more helpful
+          than JSON IMO.)
```

Comment:
Will,

I am not concerned about the size of these and the multi-line formatting helps me in reading them. I would like to find some way to use the same data structure for both the bash and python uses (these basically mirror what's done in the sour cable setup scripts under cmake/std/sems.) I have yet to see a good way to do that with either of the existing representations, so if we could do it with a third, that's a big advance.